src/hooks/renderNoteConfig/addNumberedIcons.js

```diff
@@ -24,6 +24,5 @@ export default (app, html, data) => {
       }
       $('select[name="icon"]', html).append(`<option value="${iconFilename}" ${selected}>${label}</option>`);
     }
-    $('input[name="iconSize"]').val(Math.round(game.scenes.viewed.data.grid * 0.75));
```

Comment:
If you want to keep the default size feature, but only set the size for *new* icons, you can do this:
```js
    if (!hasProperty(data, "object._id"))
        $('input[name="iconSize"]').val(Math.round(game.scenes.viewed.data.grid * 0.75));
```

Reply:
I personally am not keen on keeping the behavior as it doesn't relate to a VTTA-relevant process. I actually wonder if this whole file can't be replaced with a dependency on the module [Automatic Journal Icon Numbers](https://foundryvtt.com/packages/journal-icon-numbers/), which seems to do what this file is trying to accomplish?